src/elm/Auth.elm

```diff
@@ -189,81 +275,161 @@ view isModal shared model =
-            , onInput EnteredPrivateKey
-            , required True
-            , autocomplete False
-            ]
-            []
-        ]
-    , div []
-        [ div [ class "card__auth__pin__form" ]
-            [ viewLoginPinForm model shared ]
-        ]
-    , button
-        [ class "btn btn--primary btn--login"
-        , onClick (SubmittedLoginPrivateKey model.form)
-        ]
-        [ text_ "auth.login.submit" ]
-    , if not isModal then
-        a [ Route.href (Route.Register Nothing Nothing), class "card__auth__prompt" ]
-            [ span [] [ text_ "auth.login.register" ]
-            , span [ class "card__auth__login__mode" ] [ text_ "auth.login.registerLink" ]
-            ]
+        tr : String -> I18Next.Replacements -> String
+        tr =
+            I18Next.tr shared.translations I18Next.Curly
+
+        errors =
+            case loginStep of
+                LoginStepPassphrase ->
+                    List.map (viewFieldProblem shared Passphrase) model.problems
+
+                LoginStepPIN ->
+                    List.map (viewFieldProblem shared Pin) model.problems
 
-      else
-        text ""
+        illustration fileName =
+            img [ class "h-40 mx-auto mt-8 mb-7", src ("images/" ++ fileName) ] []
+
+        buttonClass =
+            "button button-primary min-w-full"
+
+        labelText : String -> Html msg
+        labelText tSuffix =
+            text <| t shared.translations (tSuffix ++ ".label")
+
+        pClass =
+            "text-white text-body mb-5"
+
+        viewPassphrase =
+            let
+                passphraseId =
+                    "passphrase"
+            in
+            div []
+                [ illustration "login_key.svg"
+                , p [ class pClass ]
+                    [ span [ class "text-green text-caption tracking-wide uppercase block mb-1" ] [ text ("Welcome back" ++ ",") ]
+                    , span [ class "text-white block leading-relaxed" ] [ text "Enter your 12 words that you've saved on PDF in your device" ]
+                    ]
+                , viewFieldLabel shared "auth.login.wordsMode.input" passphraseId Nothing
+                , textarea
+                    [ class "form-textarea h-19 min-w-full block"
+                    , id passphraseId
+                    , value model.form.passphrase
+                    , onInput EnteredPassphrase
+                    , required True
+                    , autocomplete False
+                    ]
+                    []
+                , div [ class "input-label pr-1 text-white font-bold mt-1 text-right" ]
+                    [ text <|
+                        let
+                            passphraseWordsCount =
+                                String.fromInt <| List.length (List.filter (not << String.isEmpty) <| String.words model.form.passphrase)
+
+                            _ =
+                                Debug.log "passphraseWordsCount"
```

Comment:
Oops... For dealing with this kind of stupid mistakes we can adopt [elm-reivew](https://github.com/jfmengels/elm-review), it may save us some time. I tried this package today and it worked well. I'll create an issue after finishing the current task with some details.